specs/vep-1243-vdk-meta-jobs/README.md

```diff
@@ -71,6 +80,17 @@ Non-goals are only features, which a contributor can reasonably assume were a go
 One example is features that were cut during scoping.
 -->
 
+### Goals
+
+* **Introduce a vdk-meta-jobs plugin**
+
+* **Provide a way to manage job dependencies**
```

Comment:
May elaborate on `job dependencies` to what? 
Are those interdependencies?

Reply:
Makes sense, changing it to interdependencies.